### PR TITLE
Remove unused code hearings worksheet

### PR DESCRIPTION
**Pull request by nolawi**
connects #2839

### CODE REVIEW

File: client/app/hearings/HearingWorksheet.jsx [hunk 1]
```diff
@@ -37,13 +37,13 @@ export class HearingWorksheet extends React.PureComponent {
 
   // TODO to move the default value to the backend
```
Comment (sharonwarner) [inline client/app/hearings/HearingWorksheet.jsx]: Can we move this TODO closer to the relevant code? We need to update the default values for the TextareaFields.

File: client/app/hearings/HearingWorksheet.jsx [hunk 2]
```diff
@@ -37,13 +37,13 @@ export class HearingWorksheet extends React.PureComponent {
...
+  onContentionsChange = (contentions) => onContentionsChange(contentions)
 
-  onPeriodsChange = (periods) => this.props.onPeriodsChange(periods)
+  onPeriodsChange = (periods) => onPeriodsChange(periods)
 
-  onEvidenceChange = (evidence) => this.props.onEvidenceChange(evidence)
+  onEvidenceChange = (evidence) => onEvidenceChange(evidence)
 
-  onCommentsChange = (comments) => this.props.onContentionsChange(comments)
+  onCommentsChange = (comments) => onContentionsChange(comments)
```
Comment (sharonwarner) [inline client/app/hearings/HearingWorksheet.jsx]: We can totally delete all of these. We call this.props.onContentionsChange which is defined at the bottom of the page.

File: client/app/hearings/HearingWorksheet.jsx [hunk 3]
```diff
@@ -299,6 +289,8 @@ export class HearingWorksheet extends React.PureComponent {
 
         <form className="cf-hearings-worksheet-form">
           <div className="cf-hearings-worksheet-data">
+
+            { /* TODO to move the default value to the backend */ }
```
Comment (sharonwarner) [inline client/app/hearings/HearingWorksheet.jsx]: I've never seen this comment syntax before. Do we need the brackets?

Comment (nolawi) [inline client/app/hearings/HearingWorksheet.jsx]: Yes, Lets see what the linter says..

**Pull request merged**